asctec_hl_gps/srv/Wgs84ToEnu.srv

```diff
@@ -0,0 +1,7 @@
+float64 lat
```

Comment:
We usually prefer having messages and service in extra packages. Here, this could be in asctec_hl_comm. This makes sure that if for some reason you cannot compile asctec_hl_gps on your machine, you can at least talk to the service on another machine.